db/create_tables.sql

```diff
@@ -1,4 +1,4 @@
-CREATE TABLE IF NOT EXISTS `audit` (
+CREATE TABLE IF NOT EXISTS `audit_records` (
   `id` BIGINT(20) NOT NULL AUTO_INCREMENT,
   `state_machine_name` VARCHAR(255) NOT NULL,
```

Comment:
why have state_machine name and state machine instance id both?
and why are both varchars ?